src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -139,12 +172,15 @@ public void initialize() {
             }
         } catch (AlfredException e) {
             logger.warning("Initialising new TeamList. "
-                           + "Problem encountered reading TeamList from storage: " + e.getMessage());
+                    + "Problem encountered reading TeamList from storage: " + e.getMessage());
             this.teamList = new TeamList();
         }
+    }
 
-        //The following try-catch block is necessary to ensure that the teamList loaded is valid
-        //and the data has not been tampered with.
+    /**
+     * Validates the team list to ensure that the data has not been tampered with.
+     */
+    private void validateTeamList() {
```

Comment:
If we initialize new entitylists, are the json files updated as well?

Reply:
I do not think so, but need @brianyenna to confirm. This PR does not change behaviour though.